generate_reports.py

```diff
@@ -3,23 +3,23 @@
 import jinja2, json, argparse
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--option", help="Select your report type(summary or default or both separeted by ,)")
+parser.add_argument("--option", help="Select your report type(default_summary, to_sponsors_summary or default_reports separeted by ,)")
```

Comment:
rename default_summary to summary
rename to_sponsors_summary utilization_summary
rename default_reports to default

Reply:
e, mude o nome dos métodos para ficar de acordo com os nomes acima